kasa/tests/test_device_type.py

```diff
@@ -1,4 +1,4 @@
-from kasa.smartdevice import DeviceType
+from kasa.iot.device import DeviceType
```

Comment:
DeviceType should be part of the main package, as it is relevant for both device families.

Reply:
It was in the main package but there were a lot of historical imports pulling it from the `Device` which I've now resolved.